Sources/FluentBenchmark/Tests/NestedModel.swift

```diff
@@ -5,7 +5,7 @@ extension FluentBenchmarker {
             UserSeed()
         ]) {
             let users = try User.query(on: self.database)
-                .filter(\.$pet, "type", .equal, User.Pet.Animal.cat)
+                .filter(\.$pet.$type,  .equal, User.Pet.Animal.cat)
```

Comment:
This can use an operator now, right? Which means we get leading-dot syntax 🙏 

```swift
.filter(\.$pet.$type == .cat)
```

Reply:
Yeah!